aries-site/src/pages/templates/global-header.mdx

```diff
@@ -143,32 +143,32 @@ for most sites.
 
 <Search /> The *Search* icon allows the user to search hpe.com.{' '}
 
-#### Shopping Cart
+#### Shopping cart
 
 <Shop /> The *Shopping Cart* icon goes to the HPE shopping cart for buy.hpe.com.
 This should only be enabled for sites which have products or solutions for sale.
 
-#### Country Selector
+#### Country selector
 
 <Language /> The country selector allows the user to go to a language-specific
 version of hpe.com. This should be disabled for most sites since the link really
 only takes you back to hpe.com instead of staying on your site.
 
-#### Sign in
+#### Sign In
```

Comment:
@taysea 
Sign in
[I know for the buttons we use the "I" in capital, but here, I think it's ok to keep the convention of the subheadings all being in sentence case. I'm not married to the idea, but just for the convention within our guide, I would suggest it in lowercase]